cmake/ecbuild_get_test_data.cmake

```diff
@@ -60,10 +60,17 @@ function( _download_test_data _p_NAME _p_DIR_URL _p_DIRLOCAL _p_CHECK_FILE_EXIST
 
       add_custom_command( OUTPUT ${_p_NAME}
         COMMENT "(curl) downloading ${_p_DIR_URL}/${_p_NAME}"
-        COMMAND ${CURL_PROGRAM} --silent --show-error --fail --output ${_p_DIRLOCAL}/${_p_NAME}
-                --retry ${ECBUILD_DOWNLOAD_RETRIES}
-                --connect-timeout ${ECBUILD_DOWNLOAD_TIMEOUT}
-                ${_p_DIR_URL}/${_p_NAME} )
+        if( ${_p_INSECURE_CURL} )
```

Comment:
To avoid duplicating the command, can you define a variable in the if statement (empty if `INSECURE_CURL` is not set) and use that in the command?

Reply:
Yes. I simplified the control sequence by introducing the `INSECURE_CURL` variable:
```cmake
        if( _p_INSECURE )
          set( INSECURE_CURL "--insecure" )
        else()
          set( INSECURE_CURL "" )
        endif()
```